scenes/user/IngameUser.gd

```diff
@@ -0,0 +1,25 @@
+extends Resource
+
+class_name IngameUser
+
+var score : float = 0
+
+func _on_good_note(note_index: int, timing_error: float):
+	# TODO : add score calculations here.
+	score += 1.0
+	changed.emit()
+
+func get_data() -> Dictionary:
+	return {
+		"score" : score,
```

Comment:
In dictionaries colons stick to the key, so "score": score. Alternatively you could use score = score, but I find it harder to read that way.